default.json

```
@@ -11,5 +11,11 @@
   ],
   "prConcurrentLimit": 5,
   "rangeStrategy": "bump",
-  "labels": ["dependencies"]
+  "labels": ["dependencies"],
+  "packageRules": [
+    {
+      "matchDatasources": ["npm"],
```

Après réflexion, la config devrait pas être spécifique à une techno, je préférais qu'on ait cette fonctionnalité pour tous nos package/lib … 
```suggestion
      "matchDatasources": ["*"],
```